examples/demo/src/reviews/ReviewCreate.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import {
+    SimpleForm,
+    Create,
+    ReferenceInput,
+    TextInput,
+    DateInput,
+    AutocompleteInput,
+    required,
+    useNotify,
+    useRedirect,
+    getRecordFromLocation,
+} from 'react-admin';
+
+import StarRatingInput from './StarRatingInput';
+import { useLocation } from 'react-router';
+
+const customerOptionRenderer = (choice: any) =>
+    `${choice.first_name} ${choice.last_name}`;
+
+const productOptionRenderer = (choice: any) => `${choice.reference}`;
+
+const ReviewCreate = () => {
+    const notify = useNotify();
+    const redirect = useRedirect();
+    const location = useLocation();
+
+    const onSuccess = (_: any) => {
+        const record = getRecordFromLocation(location);
```

Comment:
You shouldn't have to do that as `onSuccess` is called with the result of `dataProvider.create`. If our dataProvider for this example does not return it, please fix it